source/php/App.php

```diff
@@ -98,18 +111,19 @@ public function renderSearchpageMount($query) {
      *
      * @return boolean
      */
-    private static function isSearchPage() {
+    private static function isSearchPage()
+    {
 
         if (is_multisite() && (defined('SUBDOMAIN_INSTALL') && SUBDOMAIN_INSTALL === false)) {
             if (trim(strtok($_SERVER["REQUEST_URI"], '?'), "/") == trim(get_blog_details()->path, "/") && is_search()) {
                 return true;
             }
         }
 
-        if (trim(strtok($_SERVER["REQUEST_URI"], '?'), "/") == "" && is_search()) {
+        if (trim(strtok($_SERVER["REQUEST_URI"], '?'), "/") == "wptest" && is_search()) {
```

Comment:
Debug data?

Reply:
We have a small difference in our database working locally. We will make sure this one is empty when launching